test.properties

```diff
@@ -3280,7 +3283,10 @@
         **/commerce/**/*Test.java
 
     test.batch.class.names.includes[headless]=\
-        **/headless/**/*Test.java,\
+        **/headless/headless-admin-list-type-resource/**/*Test.java,\
+        **/headless/headless-admin-user-resource/**/*Test.java,\
+        **/headless/headless-admin-workflow/**/*Test.java,\
+        **/headless/headless-form-resource/**/*Test.java,\
```

Comment:
there are in Headless component in Testray? Should we get rid of them? I have Created Headless FI in order to take out ours from Headless :|

Reply:
I think I better understand now. "headless" is not going to be owned by us, but will just be a placeholder for other team tests.


And we're not moving the following because it's a mix of our tests and other product team tests?
```
  **/portal-tools-rest-builder/**/*Test.java,\
  **/portal-vulcan/**/*Test.java
```

Reply:
I think I better understand now. "headless" is not going to be owned by us, but will just be a placeholder for other team tests.


And we're not moving the following because it's a mix of our tests and other product team tests?
```
  **/portal-tools-rest-builder/**/*Test.java,\
  **/portal-vulcan/**/*Test.java
```

Reply:
correct, as Javi did not mention those two 
```
**/portal-tools-rest-builder/**/*Test.java,\
**/portal-vulcan/**/*Test.java
```
when indicating which ones are FI responsibility I didn't put them. Might will double check just in case :D

Reply:
In the end I have added vulcan and rest-builder to the [frontend] batch as well as it belongs to FI